src/features/content-explorer/content-explorer-modal-container/__tests__/ContentExplorerModalContainer.test.js

```diff
@@ -5,7 +5,7 @@ import ContentExplorerModalContainer from '../ContentExplorerModalContainer';
 
 describe('features/content-explorer/content-explorer-modal-container/ContentExplorerModalContainer', () => {
     const sandbox = sinon.sandbox.create();
-    const initialSelectedItems = { '123': { id: '123', name: 'folder123' } };
+    const initialSelectedItems = { 123: { id: '123', name: 'folder123' } };
```

Comment:
why is this change needed? wouldn't the `id` be the same as the key?

Reply:
This and similar changes were done by the formatter. I did not change this

Reply:
oh i see 👍